rust/processor/src/db/common/models/transaction_metadata_model/parquet_write_set_size_info.rs

```diff
@@ -0,0 +1,54 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+#![allow(clippy::extra_unused_lifetimes)]
+
+use crate::bq_analytics::generic_parquet_processor::{GetTimeStamp, HasVersion, NamedTable};
+use allocative_derive::Allocative;
+use aptos_protos::transaction::v1::WriteOpSizeInfo;
+use field_count::FieldCount;
+use parquet_derive::ParquetRecordWriter;
+use serde::{Deserialize, Serialize};
+
+#[derive(
+    Allocative, Clone, Debug, Default, Deserialize, FieldCount, ParquetRecordWriter, Serialize,
+)]
+pub struct WriteSetSize {
+    pub txn_version: i64,
```

Comment:
@ying-w let me know if it's missing any fields

Reply:
i thought we had talked about not using change_index but rather state_key_hash

Reply:
sync offline. this is fine to stay as it is